launcher.py

```diff
@@ -112,6 +114,9 @@ def parse_and_launch():
     viewer = None
     if args.command == 'replay':
         metalog = MetaLog(args.logfile)
+        if conf is None and args.logfile.endswith('.zip'):
+            if DEFAULT_ZIP_CONFIG in zipfile.ZipFile(args.logfile).namelist():
+                conf = Config.loads(zipfile.ZipFile(args.logfile).read(DEFAULT_ZIP_CONFIG))
```

Comment:
Ok, I misread the code.

The default file name IMHO would belong to a zip-logging facility/class/module which IIRC we do not have.